src/main/java/org/web3j/codegen/SolidityFunctionWrapper.java

```diff
@@ -72,7 +72,9 @@
             "Please use {@link " + SolidityFunctionWrapperGenerator.class.getName() +
             "} to update.\n";
 
-    public void generateJavaFiles(String contractName, String bin, String abi, String destinationDirLocation, String basePackageName) throws IOException, ClassNotFoundException {
+    public void generateJavaFiles(String contractName, String bin, String abi,
```

Comment:
Please can you start the newline after generateJavaFiles(, instead of midway through the parameter list?